apps/webapp/app/header/notifications.tsx

```diff
@@ -38,16 +38,14 @@ export const Notifications = () => {
   return (
     <Popover>
       <PopoverTrigger className='relative'>
-        <div className='relative'>
-          <div className='absolute right-[2px] top-[5px] h-[10px] w-[10px] rounded-full bg-sand'></div>
-          <Image src='/bell.svg' width={30} height={30} alt='bell' />
-        </div>
+        <div className='absolute right-[2px] top-[5px] z-10 h-[10px] w-[10px] rounded-full bg-sand'></div>
+        <ResponsiveImage src='/bell.svg' alt='Bell' className='h-[30px] w-[30px]' />
```

Comment:
```diff
- alt='Bell'
+ alt='notification bell'
```

Reply:
I just started working here! I'm adding states for this icon